blocks/rich-text/README.md

```diff
@@ -49,11 +49,11 @@ a traditional `input` field, usually when the user exits the field.
 
 ### `formattingControls: Array`
 
-*Optional.* By default, all formatting controls are present. This setting can be used to fine-tune formatting controls. Possible items: `[ 'bold', 'italic', 'strikethrough', 'link' ]`.
+*Optional.* By default, all formatting controls are enabled but hidden from the user. This setting can be used to fine-tune formatting controls. Possible items: `[ 'bold', 'italic', 'strikethrough', 'link' ]`. To show the formatting controls, set `isSelected` to `true`.
```

Comment:
I'd remove the `but hidden from the user` and `. To show the formatting controls, set `isSelected` to `true`. RichText has been updated and isSelected is not required anymore.